src/s1reader/s1_burst_id.py

```diff
@@ -113,7 +115,7 @@ def from_str(cls, burst_id_str: str):
         Parameters
         ----------
         burst_id_str : str
-            The burst ID string, e.g. "t123_456_iw1"
+            The burst ID string, e.g. "t123_00456_iw1"
```

Comment:
Just a nit pick. I thought we were using six digits in the middle part of the burst ID (i.e. `00456` in the case above). Can you check?

Reply:
ah thank you! not a nit, lol I included that to try and fix the definitely-wrong version that only had 3 digits